homeassistant/components/zwave_js/discovery.py

```diff
@@ -1058,9 +1058,24 @@ class ZWaveDiscoverySchema:
         ),
         required_values=[
             ZWaveValueDiscoverySchema(
-                command_class={
-                    CommandClass.BASIC,
-                },
+                command_class={CommandClass.BASIC},
+                type={ValueType.NUMBER},
+                property={TARGET_VALUE_PROPERTY},
+            )
+        ],
+    ),
+    # sensor for Basic CC without target
```

Comment:
Will this sensor be enabled by default? What's the use case for this sensor? In the Multisensor 6 example, that I own, it's not useful.

Reply:
I am going based on the certification task. It's really a bugfix because we currently create a light when the CC isn't controllable. Basic CC should be hidden from most devices but there was a bug and a reinterview is required to get rid of it for some scenarios

Reply:
From our tests it looks like the Multisensor 6 will create this sensor enabled by default. If that's true, we shouldn't do that. At maximum it should be created disabled by default. But it would be even better if the driver didn't send these values at all if they aren't useful.

Reply:
from this: https://github.com/zwave-js/certification-backlog/issues/7

We should enable the Basic CC entities by default.

There were two bugs:
1. Basic CC was exposed in places it shouldn't have been
2. Basic CC always came with a `targetValue` even if it couldn't be changed

Both bugs have been resolved, but they require a reinterview to resolve within the cache in zwave-js. 

If you don't agree with this approach, I would like @marcelveldt and/or @AlCalzone to weigh in as I am just implementing the requirements as written.

Reply:
Some motion sensors **only** use this to report their motion. Aeotec Multisensor 6 also does in the default configuration, and has to be manually configured to use Binary Sensor Reports instead of Basic Set.

Unfortunately Basic CC is like the name implies, basic.
There's not much we can do to find out about it's values aside from what the driver already does.

Reply:
I own that device and the motion sensor report works without adding a new sensor for basic CC. There's no need to have this sensor enabled by default.

Reply:
To be clear, the Multisensor 6 default motion sensor is using the Notification command class for the reports.

The manual configuration between basic CC and binary sensor CC is useless for most users.

Reply:
Ok bad example then. If you check the issue where users reported incorrect Basic light entities on some of their devices, you'll find some examples of simple motion sensors that only use Basic CC to report motion.

Reply:
I understand that some devices only expose the basic CC for a feature. But why should we add a sensor for devices that have another CC that represents the feature much better? I'd think that the driver can have config per device and can know if it should expose the basic CC or not. The Multisensor 6 is good example where it would be possible to not expose the basic CC since there is a better alternative in the Notification CC.

Reply:
I agree, if we are relatively certain that the basic CC mirrors another (better) sensor, then we should hide it.

Need some kind of heuristic though, like assuming that this basic sensor is the motion sensor when exposed on a device that supports reporting motion through other means.

Reply:
so what is the conclusion here? It feels like the decision is something that needs to be implemented in the driver? I think this PR is ok as is?

Reply:
I don't think we should enable this entity by default until we know it's required.

Reply:
we already enable the Basic CC light by default and this just puts some of the values into the right entity type. Do we want to reevaluate that original decision? Here's the original issue for context: https://github.com/zwave-js/certification-backlog/issues/7

Reply:
I think so. It was a terrible user experience to have loads of non functional light entities showing up for different devices. I'm trying to avoid us making the same mistake for this sensor.